### ISSUE TITLE
Background setting does not work for "Lights Out"

### ISSUE THREAD

**Issue by geyerbri** [Bl4Cc4t/GoodTwitter2#48]
After updating from 0.0.14 to 0.0.17, the "Lights Out" background setting no longer applies properly. For screenshots with "Lights Out" setting with 0.0.14, see #25.

With version 0.0.17, this is the view with "Lights Out" set:

![Capture5](https://user-images.githubusercontent.com/5496867/83543041-ca70e700-a4c9-11ea-984c-c1d47ceccd3f.PNG)

The other background settings work apparently as intended. "Default":

![Capture7](https://user-images.githubusercontent.com/5496867/83543179-099f3800-a4ca-11ea-8a82-7367a622980d.PNG)

And "Dim":

![Capture 6PNG](https://user-images.githubusercontent.com/5496867/83543231-1b80db00-a4ca-11ea-97bf-6d3395b4b670.PNG)

**Comment by Bl4Cc4t** [comment]
do you still have this issue? I cannot reproduce it on v0.0.17.
If yes, it may be fixed in v0.0.20 since I changed the color detection again

**Comment by geyerbri** [comment]
I've still got the issue, looks like my version updated to 0.0.19. 

But the stated version in the js file still still 0.0.18; I've confirmed it's 0.0.19 by checking the GH changes marked as 0.0.19 against the script running in Tampermonkey.

Hopefully, when 0.0.20 is deployed, I can confirm this has been fixed.

Thank you so much! I really do appreciate this tool and your hard work to improve it!

**Comment by Bl4Cc4t** [comment]
Seems like I forgot to adjust the version number yet again. this happens like every time >_>

**Comment by geyerbri** [comment]
After checking 0.0.20, this issue appears to persist. I even tried logging out and back in, but no change. Not sure why though, apologies. 

Perhaps this could be marked as an enhancement, since Default and Dim both appear to work?

At any rate, thank you for the hard work!

**Comment by Bl4Cc4t** [comment]
Very weird. Can you please install it again from [here](https://github.com/Bl4Cc4t/GoodTwitter2/raw/lightsout-bug/twitter.gt2eb.user.js) and tell me what output you get when clicking the theme buttons?

**Comment by geyerbri** [comment]
I installed this lightsout-bug branch version and it initially didn't work. So I tried a whole bunch of things to try and isolate, including disabling all my extensions. The only thing that ended up working was fully clearing out my browser's cache and cookies from the past hour.

Then, since that lightsout-bug branch version worked I installed your main master version and it's working properly.

So, I have absolutely no idea what in my specific case was causing it to not work, but it is. If anyone else reports, sounds like a good idea would be to close twitter, do a full cache and cookie clear, then try opening it back up, to get it working?

At any rate, looks like the master version is working for me now, so I don't think this branch's solution is even needed.
